demos/demo-jaas-webapp/src/main/webapp/login.html

```diff
@@ -1,18 +1,35 @@
 
 <html>
 <head><title>JAAS Authentication and Authorization Test</title>
-    <link rel="stylesheet" type="text/css" href="stylesheet.css"/>
+   <link rel="stylesheet" href="demo.css"/>
 </head>
 <body>
-<h1> Enter your username and password to login </h1>
-<i> Enter login=me and password=me in order to authenticate successfully</i>
-<form method="POST" action="j_security_check">
-    <b>Login: </b><input type="text" name="j_username">
-    <p>
-        <b>Password: </b><input type="password" name="j_password">
-    <p>
-        <input type="submit" value="Login"/>
-</form>
-<p>
+    <div class="topnav">
+      <a class="menu" href="http://localhost:8080/">Demo Home</a>
+      <a class="menu" href="https://github.com/eclipse/jetty.project/tree/jetty-10.0.x/demos/demo-jaas-webapp">Source</a>
```

Comment:
This one too.

Reply:
Yes, It's very very rare that these files change.